packages/node_modules/@webex/plugin-memberships/src/memberships.js

```diff
@@ -255,7 +254,7 @@ const Memberships = WebexPlugin.extend({
     };
 
     return ensureMyIdIsAvailable(this.webex)
-      .then(() => ensureConvoAvailability(this.webex)
+      .then(() => this.webex.internal.services.waitForCatalog('postauth')
```

Comment:
This should work well, note that if you're looking for a specific service, the `webex.internal.services.waitForService()` method could be used as well. Both should work based on the usecase I'm seeing though.